chart/kafka-connector/values.yaml

```diff
@@ -5,7 +5,7 @@
 # You will need to create a license named "openfaas-license" - see the
 # chart README for detailed instructions.
 
-image: ghcr.io/openfaasltd/kafka-connector:0.6.2
+image: ghcr.io/openfaasltd/kafka-connector:0.7.0
```

Comment:
Please can you rebase with the latest image?

You should be able to use the new arkade feature to try and upgrade this chart, use the `-w` flag too.